tests/test_window.py

```diff
@@ -57,8 +57,6 @@ class TestWindow(unittest.TestCase):
 
     def setUp(self):
         warnings.simplefilter("ignore", Warning)
-        self.readListFromSettingsMock = patch('ReText.window.readListFromSettings', return_value=[]).start()
-        self.writeListToSettingsMock  = patch('ReText.window.writeListToSettings').start()
```

Comment:
Now maybe you should add a mock for `globalCache`?

The purpose of these mocks is to prevent tests from messing up with user's configuration (and to make them pass when configuration is non-standard).

Reply:
I added the mock for `globalCache`.